MAINTAINERS.yml

```diff
@@ -3490,6 +3490,24 @@ Silabs Platforms:
   labels:
     - "platform: Silabs"
 
+Silabs SiM3U Platforms:
+  status: maintained
+  maintainers:
+    - rettichschnidi
+  collaborators:
+    - M1cha
```

Comment:
Could you also add the maintainers & collaborators of Silabs platforms in general (@jhedberg @asmellby and @jerome-pouiller) to collaborators list here? That'll make sure that we get added as reviewers to relevant issues and pull requests.

Reply:
done